pybamm/models/submodels/electrolyte_current.py

```diff
@@ -0,0 +1,118 @@
+#
+# Equation classes for the electrolyte current
+#
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+import pybamm
+
+import numpy as np
+
+
+class FirstOrderPotential(pybamm.BaseModel):
```

Comment:
Should the name indicate that this is specific to the StefanMaxwell electrolyte? We may have a NernstPlanck asymptotic solution aswell at some point